src/resolvers/mutations/school.js

```diff
@@ -49,6 +49,24 @@ const school = async (_, { input }, context, _info) => {
     }
 };
 
+/**
+ * @param {any} _
+ * @param {any} input
+ * @param {import('../../..').IRequestContext} context
+ * @param {any} _info
+ */
+const updateSchool = async (_, input, context, _info) => {
+    try {
+        const school = await context.prisma.updateSchool({ where: input['where'], data: input['data'] }).$fragment(utils.schoolFragment);
```

Comment:
what happens if the school does not exist?

Reply:
Throw a graphql error

Reply:
will we be able to handle this on UI? I see a scenario where a user updates something deleted by another user.

Reply:
Yes plus this mutation sisi ndio tutatumia and we will be choosing from a dropdown of existing schools